### PR TITLE
Update wrong placeholders in ar-arb translation

### PR DESCRIPTION
**Pull request by quintanilhar**
## Ticket
https://oat-sa.atlassian.net/browse/TR-3026

## Summary
Replacement characters (%s, %d, …) are not replaced in translations strings rendered.

## Dependencies
No dependencies

## Sandbox environment
TODO

### CODE REVIEW

File: locales/ar-arb/messages.po [hunk 1]
```diff
@@ -176,7 +180,7 @@ msgid "Block contains the content (stimulus) of the item such as text or image.
 msgstr "يحتوي الحظر على محتوى (التحفيز) الخاص بالبند مثل النص أو الصورة. وهو مطلوب أيضًا للتفاعلات المضمنة."
 
 msgid "Block height (%)"
-msgstr "ارتفاع الكتلة (٪)"
+msgstr "ارتفاع الكتلة (%)"
```
Comment (wazelin) [inline locales/ar-arb/messages.po:183]: This one's not a placeholder's `%` char. It probably shouldn't have been replaced.

File: locales/ar-arb/messages.po [hunk 2]
```diff
@@ -464,13 +468,13 @@ msgid "Enter tooltip target"
 msgstr "أدخل هدف تلميح الأداة"
 
 msgid "Error on item %1$s : %2$s"
-msgstr "خطأ في البند ٪ 1 $ s:٪ 2 $ s"
+msgstr "خطأ في البند %1$s : %2$s"
```
Comment (wazelin) [inline locales/ar-arb/messages.po:471]: I'm no expert at all, but just raising your attention here to the fact the first `%1$s` placeholder has been reduced to `1%`, while the second one was left as is. Was that really expected?

**Review by wazelin — CHANGES_REQUESTED** [review]

File: locales/ar-arb/messages.po [hunk 3]
```diff
@@ -779,15 +783,11 @@ msgid "Lower than or equal"
 msgstr "أقل من أو يساوي"
 
 msgid "Malformed XML[%s]:\n"
```
Comment (marpesia) [inline locales/ar-arb/messages.po:785]: I think we have missed here a line. Please, look at line 782 on the source `"%s"`. It has been removed in the translation, right?

File: locales/ar-arb/messages.po [hunk 4]
```diff
@@ -1630,16 +1629,16 @@ msgid "Unable to process your request"
...
 msgid "Unable to retrieve item : "
 msgstr "تعذر استرداد التمرين:"
 
 msgid "Unable to retrieve item : \"%s"
-msgstr "تعذر استرداد البند: \"٪ s"
+msgstr "تعذر استرداد البند: \"%s"
 
 msgid "Unable to validate %s: %s"
-msgstr "غير قادر على التحقق من صحة٪ s:٪ s"
+msgstr "غير قادر على التحقق من صحة% s:% s"
```
Comment (marpesia) [inline locales/ar-arb/messages.po:1641]: I don't think this is going to work with the space between the `%` and the `s`

File: locales/ar-arb/messages.po [hunk 5]
```diff
@@ -512,7 +516,7 @@ msgid "Fail to export item"
 msgstr "فشل تصدير البند"
 
 msgid "Failed to publish %1$s in %2$s"
-msgstr "فشل نشر٪ 1 $ s في٪ 2 $ s"
+msgstr "فشل نشر %1$s في %2$s"
```
Comment (marpesia) [inline locales/ar-arb/messages.po:519]: variable separated?

File: locales/ar-arb/messages.po [hunk 6]
```diff
@@ -1540,10 +1539,10 @@ msgid "The purpose for which the rubric is intended to be used. As a reading pas
 msgstr "الغرض من استخدام نموذج التقييم من أجله. كممر قراءة لبنود متعددة ، أو تعليمات لقسم معين ، على سبيل المثال."
 
 msgid "The QTI class \"%1$s\" located at line \"%2$d\" is currently not supported."
-msgstr "فئة QTI \"٪ 1 $ s\" الموجودة في السطر \"٪ 2 $ d\" غير مدعومة حاليًا."
+msgstr "فئة QTI \"%1$s\"  الموجودة في السطر \"%2$d\" غير مدعومة حاليًا."
```
Comment (marpesia) [inline locales/ar-arb/messages.po:1542]: variable separated?

File: locales/ar-arb/messages.po (hunk 2 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po:471]: It seems to be a glitch on the GitHub diff feature, the translation looks fine:

image

File: locales/ar-arb/messages.po (hunk 5 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po:519]: Same here: 
image

File: locales/ar-arb/messages.po (hunk 6 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po:1542]: Looks ok:
image

File: locales/ar-arb/messages.po (hunk 5 above)
Comment (marpesia) [reply locales/ar-arb/messages.po:519]: This is the branch on my local machine
image

Comment (marpesia) [reply locales/ar-arb/messages.po:519]: But on the messages_po.js
image

Comment (quintanilhar) [reply locales/ar-arb/messages.po:519]: Yes, this is an issue with text editors only. These mixed chars are not correctly rendered.

File: locales/ar-arb/messages.po (hunk 3 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po:785]: Reverted :)

File: locales/ar-arb/messages.po (hunk 1 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po]: updated

File: locales/ar-arb/messages.po (hunk 4 above)
Comment (quintanilhar) [reply locales/ar-arb/messages.po]: Fixed :)

File: locales/ar-arb/messages.po [hunk 7]
```diff
@@ -1717,41 +1716,40 @@ msgid "You may make as many association pairs as you want."
 msgstr "يمكنك عمل العديد من أزواج الربط كما تريد."
 
 msgid "You must select %1$d to %2$d choices."
-msgstr "يجب تحديد٪ 1 $ d إلى٪ 2 $ d اختيارًا."
+msgstr "يجب تحديد %1$d إلى %2$d اختيارًا."
```
[the remark this answers is not in the document]
Comment (quintanilhar) [reply locales/ar-arb/messages.po:1723]: Works as expected: 
image

**Review by marpesia — APPROVED** [review]
LGTM

File: locales/ar-arb/messages.po [hunk 8]
```diff
@@ -692,25 +696,25 @@ msgid "Invalid URL"
 msgstr "URL غير صالح"
 
 msgid "Is this choice the correct response?"
-msgstr "هل هذا الاختيار هو الرد الصحيح؟"
+msgstr "هل هذا الاختيار هو الرد الصحيح?"
 
 msgid "Is this pair the correct response?"
-msgstr "هل هذا الزوج هو الرد الصحيح؟"
+msgstr "هل هذا الزوج هو الرد الصحيح?"
```
[the remark this answers is not in the document]
Comment (quintanilhar) [reply locales/ar-arb/messages.po]: Not sure either but I have reverted it to the previous version. It's very likely that it was changed by mistake.

**Pull request merged**